MAINTAINERS.md

```diff
@@ -0,0 +1,27 @@
+# Wpt.fyi - Guidelines for Maintainers
+Thank you for your interest in helping maintain the WPT Dashboard!  Here are our PR and Issues guidelines and policies for Maintainers of the project:
+
+## Moderating Discussion
+Guidelines for issue reports and pull requests are included in this project’s `CONTRIBUTING.md` file. Ensure that contributors honor these recommendations. Close discussions in any of the following events: 
+
+1. The main description of the issue is no longer what is/was intended
+a. If the description is inaccurate or describes something other than what is discussed in the issue, the issue should be closed.
+
+2. The comment thread of an issue gets too long and contains more than just additions and clarifications.
+3. The issue is no longer relevant or it has been resolved.
+4. The issue is a request that is deemed “out of scope” for the project.
+5. The issue contains content that violates the code of conduct.
+
+Ensure contributors honor the guidelines documented in the project's [CONTRIBUTING.md](https://github.com/w3c/wptdashboard/blob/master/CONTRIBUTING.md) file.
+
+## Reviewing Pull Requests
+Pull requests will be reviewed and commented within 3 business days. There is no guaranteed time box for pull requests to be closed with or without merge; however, make an effort to do so in a reasonable time frame.
```

Comment:
We missed a bit of passive language here. Could we change this to, "Review and post comments on pull requests within 3 business days of their submission." ?